services/web-server/test/graphql/tasks_graphql_test.js

```diff
@@ -64,4 +68,81 @@ helper.secrets.mockSuite(testing.suiteName(), [], function(mock, skipping) {
       assert.equal(response.data.createTask.taskId, taskId);
     });
   });
+
+  suite('Task Subscriptions', function() {
+    helper.withMockedEventIterator();
+
+    const createSubscriptionClient = async () => {
+      return new Promise(function(resolve, reject) {
+        const subscriptionClient = new SubscriptionClient(
+          `ws://localhost:${helper.serverPort}/subscription`,
+          {
+            reconnect: true,
+          },
+          WebSocket,
+        );
+        subscriptionClient.onConnected(function() {
+          resolve(subscriptionClient);
+        });
+        subscriptionClient.onError(function(err) {
+          reject(err);
+        });
+      });
+    };
+
+    const getClient = (subscriptionClient) => {
+      const cache = new InMemoryCache();
+      const link = new WebSocketLink(subscriptionClient);
+
+      return new ApolloClient({ cache, link });
+    };
+
+    test('subscribe works', async function(){
+      // We need to create this subscription client separately so we can close it after our test
+      // Otherwise, our tests will just hang and timeout
+      let subscriptionClient = await createSubscriptionClient();
+      const client = getClient(subscriptionClient);
+      const task = helper.makeTaskDefinition();
```

Comment:
It looks like this is unused?  I see that taskGroupId is passed to the subscription on line 131, but that's not actually matched in the response (the response has taskGroupId = "subscribe-task-group-id").